packages/experimental/Badge/src/PresenceBadge/PresenceBadgeTokens.win32.ts

```diff
@@ -0,0 +1,37 @@
+import { TokenSettings, Theme } from '@fluentui-react-native/framework';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import { PresenceBadgeTokens } from './PresenceBadge.types';
+
+export const defaultPresenceBadgeTokens: TokenSettings<PresenceBadgeTokens> = (t: Theme): PresenceBadgeTokens =>
+  ({
+    borderWidth: 2,
+    borderColor: t.name === 'HighContrast' ? 'transparent' : globalTokens.color.white,
+    bottom: globalTokens.spacing.none,
+    right: globalTokens.spacing.none,
+    paddingHorizontal: globalTokens.spacing.none,
+    backgroundColor: t.name === 'HighContrast' ? globalTokens.color.black : globalTokens.color.white,
```

Comment:
Similar question here on if black is always the proper background color for the presence badge in HC

Reply:
I fixed it in another PR